cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperatorTest.java

```diff
@@ -554,6 +555,8 @@ public void testNewWithMissingHardGoals(VertxTestContext context) throws IOExcep
             })));
     }
 
+    @Disabled
+    // The generation from yaml adds duplicate "kind" property. When that is fixed, this test should be enabled
```

Comment:
Can you share more details on this?

Reply:
This command 
```KafkaRebalance kr = TestUtils.fromYamlString(yaml, KafkaRebalance.class);```
returns this object 
```
---
apiVersion: "kafka.strimzi.io/v1alpha1"
kind: "KafkaRebalance"
metadata:
  labels:
    strimzi.io/cluster: "kafka-cruise-control-test-cluster"
  name: "my-rebalance"
  namespace: "cruise-control-namespace"
spec:
  goals:
  - "CpuCapacityGoal"
  - "NetworkInboundCapacityGoal"
  - "DiskCapacityGoal"
  - "RackAwareGoal"
  - "NetworkOutboundCapacityGoal"
  - "ReplicaCapacityGoal"
  skipHardGoalCheck: false
  concurrentPartitionMovementsPerBroker: 0
  concurrentIntraBrokerPartitionMovements: 0
  concurrentLeaderMovements: 0
  replicationThrottle: 0
  unknown: "property"
kind: "KafkaRebalance"
```
Note the duplicity of `the kind property. And that is failing the validation in the tests. I did not manage to find out why is this happening.

Reply:
So, what is causing this? is there any tracking issue? `TestUtils.fromYaml` seems to use Jackson only. So I'm not sure why would this happen.

Reply:
I suspect that _something_ has changed in `CustomResource` class. The `fromYaml` calls `return mapper.readValue(yamlContent, c)` where `c` is the class of the resource. And it is what appears in the result (yaml) resouce. The version of the jackson did not change.

Reply:
Ok, fair enough. But it still applies that this is either a bug in Strimzi or in Fabric8 ... if it is in Strimzi it needs to be fixed. If in Fabric8, it needs to be reported. So you cannot just disable it, you need to create the corresponding issues.

Reply:
https://github.com/strimzi/strimzi-kafka-operator/issues/5053

Reply:
It would be good to add a comment linking the issue to the disabled tests. To make it clear why they are disabled.